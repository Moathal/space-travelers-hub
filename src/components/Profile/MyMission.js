import React from 'react';

function MyMission({ name }) {
  return (
    <h4>{name}</h4>
  );
}

export default MyMission;
