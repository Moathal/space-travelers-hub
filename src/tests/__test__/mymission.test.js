import renderer from 'react-test-renderer';
import MyMission from '../../components/Profile/MyMission';

it('renders MyMission correctly', () => {
  const tree = renderer.create(<MyMission />).toJSON();
  expect(tree).toMatchSnapshot();
});
