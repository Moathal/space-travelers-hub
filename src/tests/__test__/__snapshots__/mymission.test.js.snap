// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders MyMission correctly 1`] = `<h4 />`;
